### DISCUSSION TITLE
UE

### DISCUSSION THREAD

**Discussion by LukeKl** [community/community#19640] [Other Feature Feedback, Questions, & Ideas] [answered]
The screen will return to the top when you put a repository into a list.

**Comment by melvincwng** [comment, marked as answer]
@jinjunshi，您好，对于错误，请向 [GitHub 错误报告](https://support.github.com/contact/bug-report) 报告。 

希望这可以帮助你。
 
如果此答案有帮助，请将其标记为答案并在 GitHub 上关注我。 谢谢。

**Reply by LukeKl** [reply]
THX

**Reply by melvincwng** [reply]
没问题 ;)

**Reply by melvincwng** [reply]
@jinjunshi , 如果没有其他问题，您可以“标记为答案”以结束此讨论。 谢谢你。

**Reply by Martinezz420** [reply]
GREAT

**Reply by macmac0726171992** [reply]
Amazing thanks

**Reply by Tds33** [reply]
Hi

**Comment by Setr43** [comment]
Cjmhln

**Reply by Setr43** [reply]
https://github.community/t/how-to-remove-github-mobile-from-2fa-list/259797?u=setr43

**Comment by Setr43** [comment]
> Cjmhln

**Comment by Setr43** [comment]
BUG-b95986992202b5a636bf532d71a2822406a145ec47d8f5394a5d5fb40eec85e0-20220703180915Z-f6336e8f9f898f0c

**Comment by Setr43** [comment]
https://github.com/orgs/github-community/discussions/19640#discussioncomment-3073693

**Comment by macmac0726171992** [comment]
> The screen will return to the top when you put a repository into a list.
